.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -21,13 +21,16 @@ jobs:
           - kernel
           - kernel-dragonball-experimental
           - kernel-tdx-experimental
+          - kernel-sev-experimental
           - nydus
           - qemu
           - qemu-tdx-experimental
+          - qemu-snp-experimental
```

Comment:
Is SNP support already merged in an upstream QEMU?
If so, it doesn't have to be an "-experimental" QEMU.